go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -537,7 +537,7 @@ func (be *BuiltinBackupEngine) ExecuteRestore(
 
 	if err := be.restoreFiles(context.Background(), cnf, bh, bm.FileEntries, bm.TransformHook, !bm.SkipCompress, restoreConcurrency, hookExtraEnv, logger); err != nil {
 		// don't delete the file here because that is how we detect an interrupted restore
-		return zeroPosition, err
+		return zeroPosition, fmt.Errorf("failed to restore files: %v", err)
```

Comment:
for consistency all uses of `fmt.Errorf` should be replaced with `vterrors.Errorf` or `vterrors.Wrapf`.

Reply:
Done.